taichi/backends/opengl/opengl_api.cpp

```diff
@@ -293,9 +301,12 @@ struct CompiledKernel {
         num_groups(num_groups_),
         used(used_) {
     glsl->link();
-    TI_DEBUG("source of kernel [{}] * {}:\n{}", kernel_name, num_groups,
-             kernel_source_code);
-#ifdef _GLSL_DEBUG
+    if (1 &&  // !ti.cfg.opengl.print_accessor_glsl
```

Comment:
Remove debug info? Or maybe make it less ad-hoc.

Reply:
Not debug info, it's a TODO. Will add that to ti.cfg options later. Will remove this now.